packages/react-router-native/__tests__/Link-test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import ReactTestUtils from 'react-addons-test-utils'
```

Comment:
Sorry to dredge this up, but this needs to be updated now that the react addons are deprecated. Should just be `import ReactTestUtils from 'react-dom/test-utils'`.